Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -46,6 +46,12 @@ $sourcePath = $sourcePath.Trim('"')
 $storageAccount = $storageAccount.Trim()
 $containerName = $containerName.Trim().ToLower()
 
+# Normalize admin username
```

Comment:
Can we move this in the try block where we handle Azure VM copy. This check is not required for Azure blob copy.